src/sas/qtgui/Calculators/media/sld_calculator_help.rst

```diff
@@ -2,57 +2,126 @@
+This SLD calculator utilizes the periodictable python package\ :sup:`1`, which is an extensible periodic table populated with values useful for neutron and x-ray experiments.
 
-The neutron scattering length density (SLD, $\beta_N$) is defined as
+User Inputs
+----------------------------
+**Molecular Formula** (required)
+    This field defines the material for which you are calculating the SLD. For more guidance on how to enter molecules, biomolecules, and more complex mixtures scroll down to "Specifying Materials or Mixtures in the Molecular Formula Field" section.
+
+**Mass Density** |g/cm^3| (required)
+    This field defines the density of the material for which you are calculating the SLD of your material. Density uncertainty is likely the largest source of error in the SLD calculator.
 
-.. math::
+**Neutron Wavelength** (|Ang|) (optional)
+    Wavelength is used to calculate the scattering cross-section and 1/e length. It is not required for SLD calculation. If left blank wavelength is assumed to be 6 |Ang|.
 
-  \beta_N = (b_{c1} + b_{c2} + ... + b_{cn}) / V_m
+**X-ray Wavelength** (|Ang|) (required)
+    X-ray scattering length density is energy dependent thus wavelength is a required field for x-ray SLD calculation.
+
+Calculator Output
+----------------------------
+Neutron SLD
+(|Ang^-2|)
 
-where $b_{ci}$ is the bound coherent scattering length of ith of n atoms in a molecule
-with the molecular volume $V_m$.
+X-ray SLD
+(|Ang^-2|)
 
-.. ZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZ
+Neutron Incoherent Cross-section (cm\ :sup:`-1`)
 
-Specifying the Compound Name
+Neutron Absorption Cross-Section (cm\ :sup:`-1`)
+
+Neutron 1/e length (cm)
+
+Specifying Materials or Mixtures in the Molecular Formula Field
 ----------------------------
+**Molecular formulas** can be entered intuitively with atoms being represented by their atomic symbol. For example, let us consider calcium carbonate (CaCO\ :sub:`3`):
+
+    CaCO3
+
+**Multipart species** can be constructed by as a single molecular formula, or by separating the components with a + or a space. For example let us consider the hexahydrate of calcium carbonate, ikaite, (CaCO\ :sub:`3` |cdot| 6(H\ :sub:`2`\O)) which can be denoted as:
+
+    CaCO3(H2O)6
+
+    CaCO3 6H2O
+
+    CaCO3+6H2O
+
+**Isotopes** are represented as the element symbol followed by its mass number in square brackets, for example C[12] and C[13]. A special exception to this are isotopes of hydrogen which an also be represented as H = H[1], D = H[2], and T=H[3].
+
+**Isotopic Mixtures** can be entered in several ways. Take, for example a mixture of 30% H2O and 70% D2O (mole fraction). It can be entered as:
+
+    H3D7O5
+
+    H[1]3H[2]7O5
+
+    3H2O+7D2O
+
+For isotopic substitution you must also adjust the density for your mixture.
```

Comment:
This is an important statement that could easily be missed if it is not highlighted (bold/italic/etc.)

Reply:
Agreed. Italicized.